Code/Numerics/Alignment/Wrap/rdAlignment.cpp

```diff
@@ -23,6 +23,24 @@ namespace python = boost::python;
 
 namespace RDNumeric {
 namespace Alignments {
+
+class PointVectManager {
+ public:
+  PointVectManager() = default;
+  PointVectManager(const PointVectManager &rhs) = delete;
+  PointVectManager &operator=(const PointVectManager &rhs) = delete;
```

Comment:
don't you also need to delete the move ctor and move-assignment operator?

Reply:
It's not mandatory to declare them, and it wouldn't cause an issue. I deleted the copy construction/assignment to prevent multiple instances to each pointer in the vector (as the duplicated managers would trigger multiple deletions in the destructor). This is not a problem with move semantics, since there's not duplication of the pointers.